src/Accelerators/NNPA/Compiler/NNPACompilerOptions.cpp

```diff
@@ -29,7 +29,8 @@ llvm::cl::opt<NNPAEmissionTargetType> nnpaEmissionTarget(
 llvm::cl::opt<bool> nnpaClipToDLFloatRange("nnpa-clip-to-dlfloat-range",
     llvm::cl::desc("Clip CPU tensors to dlfloat range before stickification to "
                    "avoid out-of-range. Only clip Softmax inputs at this "
-                   "moment. Default is true."),
+                   "moment. Default is true. This option will be removed and "
+                   "replaced by --nnpa-saturation in the future."),
```

Comment:
Clip would be redundant only if the softmax is preceded by a stick-with-saturation. Maybe that is the case in the models we see... or maybe the saturation of the entire model transitively prevent the values to be out of bound once they reach softmax

Reply:
Thanks for the suggestion! Will investigate more.